src/fr/nif.ts

```diff
@@ -50,10 +50,20 @@ const impl: Validator = {
     if (value.length !== 13) {
       return { isValid: false, error: new exceptions.InvalidLength() };
     }
-    if (!strings.isdigits(value)) {
+    const startsWithOneOf0123 = [0, 1, 2, 3].includes(
```

Comment:
Technically this would end up being an `InvalidateComponent` rather than `InvalidFormat` error, but that's a minor difference.